tedge_config/src/tedge_config.rs

```diff
@@ -1,6 +1,9 @@
 use crate::*;
 
-const DEFAULT_ROOT_CERT_PATH: &str = "/etc/ssl/certs";
+const DEFAULT_DEVICE_CERT_PATH: &str = "/etc/ssl/certs/tedge-certificate.pem";
+const DEFAULT_DEVICE_KEY_PATH: &str = "/etc/ssl/certs/tedge-private-key.pem";
+const DEFAULT_AZURE_ROOT_CERT_PATH: &str = "/etc/ssl/certs";
+const DEFAULT_C8Y_ROOT_CERT_PATH: &str = "/etc/ssl/certs";
```

Comment:
The main issue I see here is that there are two set of defaults.
* One at the system level - when running as `sudo tedge`
* One at the user level - when running as `tedge` - and depending on $HOME.
* And even a 3rd ones for the tests where the base directory must be independent of $HOME.

So I would expect two functions to build these defaults
* at the system level
* at the user level in a given dir.